app/service/cdo.js

```diff
@@ -91,14 +92,46 @@ class CdoService {
       await sendUpdateToAudit(EXEMPTION, preChanged, postChanged, user)
     }
 
-    cdoTaskList.addInsuranceDetails(insuranceDetails.insuranceCompany, insuranceDetails.insuranceRenewal, sendEvent)
+    cdoTaskList.recordInsuranceDetails(insuranceDetails.insuranceCompany, insuranceDetails.insuranceRenewal, sendEvent)
 
     const returnedCdoTaskList = await this.cdoRepository.saveCdoTaskList(cdoTaskList)
 
     const { insuranceCompany, insuranceRenewal } = returnedCdoTaskList.cdoSummary
 
     return { insuranceCompany, insuranceRenewal }
   }
+
+  /**
+   *
+   * @param {string} cdoIndexNumber
+   * @param {{microchipNumber: string}} microchip
+   * @param user
+   * @return {Promise<import('../data/domain').CdoTaskList>}
+   */
+  async recordMicrochipNumber (cdoIndexNumber, microchip, user) {
+    const microchipNumber = microchip.microchipNumber
+
+    const cdoTaskList = await this.cdoRepository.getCdoTaskList(cdoIndexNumber)
+    const preMicrochipNumber = cdoTaskList.cdoSummary.microchipNumber
+
+    const callback = async () => {
+      const preAudit = {
+        index_number: cdoIndexNumber,
+        microchip1: preMicrochipNumber ?? null
```

Comment:
How have we previously audited microchip number changes? I might be wrong but I thought the field names were microchipNumber and microchipNumber2, so we should try to keep the same filed name(s) here